microcosm_elasticsearch/tests/test_searching.py

```diff
@@ -65,10 +64,10 @@ def test_count_single_type(self):
         with self.player_store.flushing():
             self.player_store.create(self.steph)
 
-        index = PersonSearchIndex.for_only(self.graph, self.graph.example_index, Player)
+        index = PersonSearchIndex(self.graph, self.graph.example_index, Player)
```

Comment:
I removed the `for_only` `@classmethod`; it no longer has the semantics it used to (because we've changed filtering) and wasn't used.